incubator/model-json/src/main/java/io/aklivity/zilla/runtime/model/json/internal/JsonModelHandler.java

```diff
@@ -76,8 +76,15 @@ protected final boolean validate(
         try
         {
             JsonProvider provider = supplyProvider(schemaId);
-            in.wrap(buffer, index, length);
-            provider.createReader(in).readValue();
+            if (provider != null)
+            {
+                in.wrap(buffer, index, length);
+                provider.createReader(in).readValue();
+            }
+            else
+            {
+                status = false;
+            }
```

Comment:
Try to avoid using `else` clauses to assign boolean variables like this.

Instead, aim to take an approach that would also work for multiple conditions discovered incrementally.

```suggestion
            status &= provider != null;

            if (status)
            {
                in.wrap(buffer, index, length);
                provider.createReader(in).readValue();
            }
```